### ISSUE TITLE
Stop verbosity of search

### ISSUE THREAD

**Issue by 23pointsNorth** [SimonBlanke/Hyperactive#70]
When running an optimization, the `tqdm` progress bar is started regardless of the way the optimization is started. 

Is there a way to decrease the verbosity?

**Comment by SimonBlanke** [comment]
I fixed this in 9afa1fe644eb6f3d37d3e3adda0b3029ca9f5e18, did some basic manual testing and released the fix in v4.4.2.
This version should be live in a few minutes.

You can close this issue if the fix was successful for your code.

until next time ;-)

**Comment by 23pointsNorth** [comment]
The current state is nearly there! However, setting 
```
hyper = Hyperactive(verbosity=False)
# or
hyper = Hyperactive(verbosity=[])
```
still results in an empty line being printed.
